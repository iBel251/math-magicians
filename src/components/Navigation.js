import { Link } from 'react-router-dom';

export default function Navigation() {
  return (
    <div className="navigation">
      <h1>Math Magicians</h1>
      <nav>
        <Link to="/Home" className="home">Home</Link>
        <Link to="/Calculator" className="calc">Calculator</Link>
        <Link to="/Quotes" className="quotes">Quotes</Link>
      </nav>
    </div>
  );
}
