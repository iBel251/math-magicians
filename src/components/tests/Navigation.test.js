import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navigation from '../Navigation';

const Mocked = () => (
    <BrowserRouter>
        <Navigation />
    </BrowserRouter>
)

test('renders correctly', () => {
  const tree = renderer.create(<Mocked />).toJSON();
  expect(tree).toMatchSnapshot();
});
