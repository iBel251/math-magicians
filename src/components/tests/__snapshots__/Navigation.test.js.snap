// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="navigation"
>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <a
      className="home"
      href="/Home"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="calc"
      href="/Calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="quotes"
      href="/Quotes"
      onClick={[Function]}
    >
      Quotes
    </a>
  </nav>
</div>
`;
